src/ripple/protocol/impl/STObject.cpp

```diff
@@ -35,7 +35,7 @@ STObject::~STObject()
 #endif
 }
 
-STObject::STObject(STObject&& other)
+STObject::STObject(STObject&& other) noexcept
     : STBase(other.getFName())
```

Comment:
`noexcept(other.getFName()` is false.

Reply:
The `STBase` constructor taking `other.getFName()` is not noexcept.

Reply:
Another good catch. Fixed in f1e59f075